server/auth/routes.js

```diff
@@ -17,6 +17,8 @@ router.post("/login", async (req, res, next) => {
   console.log(`Login attempt ${email}`);
   try {
     const user = await db.getUserByEmail(email);
+    const languages = await db.getUserLanguages(email);
```

Comment:
I think it would be better if you get the user languages after you have verified that the user exists (there is an `if` just below that does that. 

Then, you can use the user id to fetch the user languages instead of the email (check comment on database/users.js)

Reply:
done